force-app/tests/Integration Recipes/CalloutRecipes_Tests.cls

```diff
@@ -139,7 +139,7 @@ private inherited sharing class CalloutRecipes_Tests {
         Integer statusCode = callouts.httpDeleteCalloutToSecondOrg(contact.Id);
         Test.stopTest();
 
-        System.assert(statusCode == 200, 'Expected a successful delete');
+        System.Assert.isTrue(statusCode == 200, 'Expected a successful delete');
```

Comment:
Change to `areEqual`

Reply:
I won't spam you with comments on this file but this remark applies to all `statusCode` tests further down.